playtron-os-scripts/playtron-os-scripts.spec

```diff
@@ -15,23 +15,30 @@ Requires: cloud-utils-growpart fio
 %setup -q -c
 
 %install
-mkdir -p %{buildroot}/etc/sysctl.d/
-cp -Rv playtron-os-scripts-%{version}/etc/sysctl.d/* %{buildroot}/etc/sysctl.d/
-mkdir -p %{buildroot}/usr/bin/
-cp -Rv playtron-os-scripts-%{version}/bin/* %{buildroot}/usr/bin/
-mkdir -p %{buildroot}/usr/lib/systemd/system/
-cp -Rv playtron-os-scripts-%{version}/lib/systemd/system/* %{buildroot}/usr/lib/systemd/system/
+cp -Rv playtron-os-scripts-%{version}/etc/ %{buildroot}/
+
+mkdir -p %{buildroot}/usr/
+cp -Rv playtron-os-scripts-%{version}/bin/ %{buildroot}/usr/
+cp -Rv playtron-os-scripts-%{version}/lib/ %{buildroot}/usr/
+cp -Rv playtron-os-scripts-%{version}/share/ %{buildroot}/usr/
+
 mkdir -p %{buildroot}/usr/share/licenses/playtron-os-scripts/
-cp playtron-os-scripts-%{version}/LICENSE %{buildroot}/usr/share/licenses/playtron-os-scripts/LICENSE
+cp playtron-os-scripts-%{version}/LICENSE %{buildroot}/usr/share/licenses/playtron-os-scripts/
 
 %files
```

Comment:
Looks like we need to add `/usr/lib/udev/rules.d/50-lenovo-legion-controller.rules` here to avoid these warnings.

```
Checking for unpackaged file(s): /usr/lib/rpm/check-files /home/ekultails/rpmbuild/BUILDROOT/playtron-os-scripts-0.4.0-1.fc38.x86_64
error: Installed (but unpackaged) file(s) found:
   /usr/lib/udev/rules.d/50-lenovo-legion-controller.rules

RPM build errors:
    Installed (but unpackaged) file(s) found:
   /usr/lib/udev/rules.d/50-lenovo-legion-controller.rules
```

Reply:
Oops. Nice catch. In my testing I was building one commit before the lenovo controller rules were added.

Fixed.